ci-scripts/JenkinsFile-OAI-Container-GitHub

```diff
@@ -98,7 +98,9 @@ pipeline {
             try {
               // Check if the pull request has files that will impact MME behavior
               // If so, we will run the OAI pipeline
-              sh 'python3 ci-scripts/check_pr_modified_files_for_oai_pipeline.py'
+              // For security reasons, we retrieve from a trusted forked repository
+              sh 'wget https://raw.githubusercontent.com/rdefosse/magma/trusted-oai-ci-pipeline/ci-scripts/check_pr_modified_files_for_oai_pipeline.py -o ci-scripts/ci-check_pr_modified_files_for_oai_pipeline.py'
```

Comment:
The overall looks good however I think the source should be user agnostic. (we have a dedicated magma repo for that)

Reply:
@tmdzk 

I can pass both (`userspace` and `branch-name`) as parameters.

Is it OK with you?

Raphael

Reply:
Then once this PR is merged, I can set the parameters to (`magma` and `master`).